doc/misc/npm-scripts.md

```diff
@@ -119,6 +119,8 @@ executing the scripts.  So, if your package.json has this:
 then you could run `npm start` to execute the `bar` script, which is
 exported into the `node_modules/.bin` directory on `npm install`.
 
+This is especially important for proper Windows support.
```

Comment:
The thing is that people do `./node_modules/.bin/foo bar` and fails in scripts whereas `foo bar` works fine everywhere.